File: packages/aws-cdk/lib/init-templates/sample-app/csharp/src/%name.PascalCased%/%name.PascalCased%Stack.template.cs
using Amazon.CDK;
using Amazon.CDK.AWS.IAM;
using Amazon.CDK.AWS.SNS;
using Amazon.CDK.AWS.SNS.Subscriptions;
using Amazon.CDK.AWS.SQS;

namespace %name.PascalCased%
{
    public class %name.PascalCased%Stack : Stack
    {
        public %name.PascalCased%Stack(Construct parent, string id, IStackProps props) : base(parent, id, props)
        {
             // The CDK includes built-in constructs for most resource types, such as Queues and Topics.
            var queue = new Queue(this, "MyFirstQueue", new QueueProps
            {
                VisibilityTimeout = Duration.Seconds(300)
            });

            var topic = new Topic(this, "MyFirstTopic", new TopicProps
            {
                DisplayName = "My First Topic Yeah"
            });

            topic.AddSubscription(new SqsSubscription(queue, null));

            // You can also define your own constructs and use them in your stack.
            %name.PascalCased%Construct hello = new %name.PascalCased%Construct(this, "Buckets", new %name.PascalCased%ConstructProps()
            {
                BucketCount = 5
            });

            // Create a new user with read access to the HelloConstruct resource.
            User user = new User(this, "MyUser", new UserProps());
            hello.GrantRead(user);
        }
    }
}

File: packages/aws-cdk/lib/init-templates/sample-app/java/src/main/java/com/myorg/%name.PascalCased%Stack.template.java
package com.myorg;

import software.amazon.awscdk.core.Construct;
import software.amazon.awscdk.core.Duration;
import software.amazon.awscdk.core.Stack;
import software.amazon.awscdk.core.StackProps;
import software.amazon.awscdk.services.sns.Topic;
import software.amazon.awscdk.services.sns.subscriptions.SqsSubscription;
import software.amazon.awscdk.services.sqs.Queue;

public class %name.PascalCased%Stack extends Stack {
    public %name.PascalCased%Stack(final Construct parent, final String id) {
        this(parent, id, null);
    }

    public %name.PascalCased%Stack(final Construct parent, final String id, final StackProps props) {
        super(parent, id, props);

        final Queue queue = Queue.Builder.create(this, "%name.PascalCased%Queue")
                .visibilityTimeout(Duration.seconds(300))
                .build();

        final Topic topic = Topic.Builder.create(this, "%name.PascalCased%Topic")
            .displayName("My First Topic Yeah")
            .build();

        topic.addSubscription(new SqsSubscription(queue));
    }
}
